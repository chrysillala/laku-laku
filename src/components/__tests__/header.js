import React from 'react'
import renderer from 'react-test-renderer'

import Header from '../header'

describe("Header", () => {
  it("header renders correctly", () => {
    const tree = renderer
      .create(<Header metaTitle="This is meta title" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})