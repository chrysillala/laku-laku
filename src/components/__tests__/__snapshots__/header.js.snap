// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header header renders correctly 1`] = `
<header
  css={
    Object {
      "map": undefined,
      "name": "17cokho",
      "next": undefined,
      "styles": "margin-bottom: 1.5rem;",
      "toString": [Function],
    }
  }
>
  <a
    css={
      Object {
        "map": undefined,
        "name": "1v0dam5",
        "next": undefined,
        "styles": "
        text-shadow: none;
        background-image: none;
      ",
        "toString": [Function],
      }
    }
    href="/"
  >
    <h3
      css={
        Object {
          "map": undefined,
          "name": "1qc4l4w",
          "next": undefined,
          "styles": "display: inline;",
          "toString": [Function],
        }
      }
    >
      This is meta title
    </h3>
  </a>
  <ul
    css={
      Object {
        "map": undefined,
        "name": "bqpj6q",
        "next": undefined,
        "styles": "listStyle: none; float: right;",
        "toString": [Function],
      }
    }
  >
    <li
      css={
        Object {
          "map": undefined,
          "name": "19fma4n",
          "next": undefined,
          "styles": "
    display: inline-block;
    margin-right: 1rem;
  ",
          "toString": [Function],
        }
      }
    >
      <a
        href="/"
      >
        Home
      </a>
    </li>
    <li
      css={
        Object {
          "map": undefined,
          "name": "19fma4n",
          "next": undefined,
          "styles": "
    display: inline-block;
    margin-right: 1rem;
  ",
          "toString": [Function],
        }
      }
    >
      <a
        href="/about/"
      >
        About
      </a>
    </li>
    <li
      css={
        Object {
          "map": undefined,
          "name": "19fma4n",
          "next": undefined,
          "styles": "
    display: inline-block;
    margin-right: 1rem;
  ",
          "toString": [Function],
        }
      }
    >
      <a
        href="/contact/"
      >
        Contact
      </a>
    </li>
  </ul>
</header>
`;
